addons/common/functions/fnc_blurScreen.sqf

```diff
@@ -10,6 +10,8 @@
 
 #include "script_component.hpp"
 
+if !(hasInterface) exitWith {};
```

Comment:
I don't know who came up with the idea to move the exclamation mark outside of the parentheses, but it looks ugly and wrong to me.

`if (!hasInterface) exitWith {};`